code/_onclick/hud/screen_objects.dm

```diff
@@ -363,7 +363,7 @@
 /obj/screen/storage/Click(location, control, params)
 	if(world.time <= usr.next_move)
 		return TRUE
-	if(usr.stat || usr.IsUnconscious() || usr.IsKnockdown() || usr.IsStun())
+	if(usr.incapacitated())

```

Comment:
Keep in mind incapacitated also checks for restraints. This will prevent users from being able to click things if they're in a grab unless you set ignore_grab to TRUE